src/Common/src/Interop/Windows/mincore/Interop.GetConsoleTitle.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Console_L1, CharSet = CharSet.Unicode, SetLastError = true, EntryPoint = "GetConsoleTitleW")]
+        private static extern int GetConsoleTitle(StringBuilder title, int nSize);
```

Comment:
There's no need to use a StringBuilder here.  I realize Console.Title is rare to use so perf isn't as big a deal, and that this is existing code you're just porting, so I'm just highlighting that StringBuilder marshaling is actually relatively expensive.

(If you keep it as StringBuilder, then at least annotate it as [Out].)